src/placeholder/solvers/base.py

```diff
@@ -1,86 +1,31 @@
+import numpy as np
+import scipy.optimize as sciopt
```

Comment:
Should we rename this file `scipy_opt` instead of `base`? Or were you thinking we would put a bunch of standard solvers in here, like AD / IPOPT?

Reply:
yeah, I was thinking maybe there would be other solvers like IPOPT